pkg/rekor/rekor.go

```diff
@@ -188,9 +189,10 @@ func (c *Client) Verify(ctx context.Context, commitSHA string, cert *x509.Certif
 	return e, cosign.VerifyTLogEntryOffline(ctx, e, c.publicKeys)
 }
 
-// extractCerts is taken from cosign's cmd/cosign/cli/verify/verify_blob.go.
+// extractCertsForHash is taken from cosign's
+// cmd/cosign/cli/verify/verify_blob.go.
 // TODO: Refactor this into a shared lib.
-func extractCerts(e *models.LogEntryAnon) ([]*x509.Certificate, error) {
+func extractCertsForHash(e *models.LogEntryAnon, hash string) ([]*x509.Certificate, error) {
```

Comment:
```suggestion
// This function extracts the data hash and certs from a given LogEntryAnon.
func extractData(e *models.LogEntryAnon) (string, []*x509.Certificate, error) {
```

The original intent for this func was to be a generic "get certs from log entry", and make the policy decisions above where there is more context. We should follow a similar pattern here instead of plumbing through the hash and splitting where the hash vs pk policy checks are being done.

Reply:
done!